files/zh-cn/web/api/element/insertadjacentelement/index.md

```diff
@@ -1,44 +1,44 @@
 ---
 title: Element.insertAdjacentElement()
 slug: Web/API/Element/insertAdjacentElement
+l10n:
+  sourceCommit: bbf7f25f9cf95fb154e2740a9fdc9c02818981bf
 ---
 
 {{APIRef("DOM")}}
 
-**`insertAdjacentElement()`** 方法将一个给定的元素节点插入到相对于被调用的元素的给定的一个位置。
+{{domxref("Element")}} 接口的 **`insertAdjacentElement()`** 方法将一个给定的元素节点插入到相对于当前元素的指定位置。
 
 ## 语法
 
+```js-nolint
+insertAdjacentElement(position, element)
 ```
-element.insertAdjacentElement(position, element);
-```
-
-### 参数
 
 - position
 
-  - : A {{domxref("DOMString")}} 表示相对于该元素的位置；必须是以下字符串之一：
+  - : 表示相对于 `targetElement` 的位置的字符串；必须是以下字符串之一（大小写不敏感）：
 
-    - `'beforebegin'`: 在该元素本身的前面。
-    - `'afterbegin'`:只在该元素当中，在该元素第一个子孩子前面。
-    - `'beforeend'`:只在该元素当中，在该元素最后一个子孩子后面。
-    - `'afterend'`: 在该元素本身的后面。
+    - `'beforebegin'`：`targetElement` 之前。
+    - `'afterbegin'`：`targetElement` 内部的第一个子节点之前。
+    - `'beforeend'`：`targetElement` 内部的最后一个子节点之后。
+    - `'afterend'`：`targetElement` 之后。
 
 - element
   - : 要插入到树中的元素。
 
 ### 返回值
 
-插入的元素，插入失败则返回 null.
+插入的元素，插入失败则返回 `null`。
 
 ### 异常
 
-| 异常          | 说明                           |
-| ------------- | ------------------------------ |
-| `SyntaxError` | 插入的位置是一个无法识别的值。 |
-| `TypeError`   | 插入的元素不是一个有效元素。   |
+- `SyntaxError` {{domxref("DOMException")}}
+  - 如果指定的 `position` 不是可识别的值，则抛出该异常。
+- {{jsxref("TypeError")}}
+  - 如果指定的 `element` 不是有效的元素，则抛出该异常。
```

Comment:
Fixed in upstream, please follow the changes:

```suggestion
  - : 如果指定的 `position` 不是可识别的值，则抛出该异常。
- {{jsxref("TypeError")}}
  - : 如果指定的 `element` 不是有效的元素，则抛出该异常。
```

Reply:
Okay, sorry, I just saw the email now.